pallets/identity/src/lib.rs

```diff
@@ -141,6 +141,12 @@ pub struct Claim2ndKey {
     pub scope: Option<Scope>,
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, PartialOrd, Ord)]
+pub struct Claim2ndKeyOld {
```

Comment:
Btw, you can derive this type via `#[derive(Migrate)]` but you need `use IdentityId as ScopeOld;`.
Perhaps we should extend the macro to handle renaming customization via e.g.,
```rust
#[migrate(New = Old, ...)]
```

Reply:
@maxsam4 Should we perhaps capture this as a separate story too?